asset-registry/Cargo.toml

```diff
@@ -8,21 +8,23 @@ authors = ["Interlay Ltd, etc"]
 edition = "2021"
 
 [dependencies]
+log = { version = "0.4.20", default-features = false }
 scale-info = { version = "2.1.2", default-features = false, features = ["derive"] }
 serde = { version = "1.0.136", optional = true }
 codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["max-encoded-len"] }
 
 # substrate
 frame-support = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v1.0.0" }
 frame-system = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v1.0.0" }
+sp-io = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v1.0.0" }
 sp-runtime = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v1.0.0" }
 sp-std = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v1.0.0" }
 
 # polkadot
 pallet-xcm = { git = "https://github.com/paritytech/polkadot", default-features = false , branch = "release-v1.0.0" }
-xcm = { git = "https://github.com/paritytech/polkadot", default-features = false , branch = "release-v1.0.0" }
-xcm-builder = { git = "https://github.com/paritytech/polkadot", default-features = false , branch = "release-v1.0.0" }
-xcm-executor = { git = "https://github.com/paritytech/polkadot", default-features = false , branch = "release-v1.0.0" }
+xcm = { package = "staging-xcm", git = "https://github.com/paritytech/polkadot", default-features = false , branch = "release-v1.0.0" }
+xcm-builder = { package = "staging-xcm-builder", git = "https://github.com/paritytech/polkadot", default-features = false , branch = "release-v1.0.0" }
+xcm-executor = { package = "staging-xcm-executor", git = "https://github.com/paritytech/polkadot", default-features = false , branch = "release-v1.0.0" }
```

Comment:
Same comment here regarding the package. Also, I see you are using the version number 1.0.0 here, it should be the same as in `Cargo.dev.toml` (1.1.0 in this case).

Another error I see is that you are pointing to the upstream branch, you should point to our fork's branch.

Reply:
I have answered these 2 comments above.